packages/generators/generators/lib/templates/js/plugin-package.json.hbs

```diff
@@ -8,7 +8,25 @@
     "kind": "plugin",
     "displayName": "{{titleCase pluginName }}"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@strapi/design-system": "^1.6.3",
+    "@strapi/helper-plugin": "^4.6.0",
+    "@strapi/icons": "^1.6.3",
+    "axios": "^1.2.2",
+    "prop-types": "^15.7.2"
+  },
+  "devDependencies": {
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-router-dom": "^5.3.4",
+    "styled-components": "^5.3.6"
```

Comment:
Why are these dev-dependencies?

Reply:
Because if someone publishes a plugin they shouldn't be installing their own versions of these libs they should be peer deps otherwise it creates errors like "Cannot find X", especially with react-router-dom.

While I _know_ the current plugin setup wouldn't let this happen (because we force resolution), we should be pre-emptive and support best practice here imo

Reply:
Right, that makes sense. Thanks!